src/sentry/tasks/post_process.py

```diff
@@ -533,12 +533,23 @@ def post_process_group(
 def run_post_process_job(job: PostProcessJob):
     group_event = job["event"]
     issue_category = group_event.group.issue_category
+
     if group_event.group.issue_type.type_id in get_group_types_by_category(
         GroupCategory.PROFILE.value
     ) and not features.has(
         "organizations:profile-blocked-main-thread-ppg", group_event.group.organization
     ):
         return
+
+    if (
+        group_event.group.issue_type.type_id
+        in get_group_types_by_category(GroupCategory.PERFORMANCE.value)
+        and options.get("performance.issues.send_to_issues_platform", False)
+        and group_event.project.get_option(
+            "sentry:performance_issue_send_to_issues_platform", False
+        )
+    ):
+        return
```

Comment:
I think we want to remove this check. Passing `skip_consume=True` to `eventstream` means that we'll already skip sending these to `post_process_group`. But if we have this check here, then issue alerts will not fire for performance issues at all

Reply:
ah okay. removed!